src/services/TechRecordsService.ts

```diff
@@ -365,6 +366,39 @@ class TechRecordsService {
     }
   }
 
+  public async managePlatesGeneration(techRecord: ITechRecord) {
+    let platesGenerated = false;
+    if (techRecord.plates) {
+      for (const plate of techRecord.plates) {
+        if (!plate.plateSerialNumber) {
+          if (!plate.plateIssueDate || !plate.plateIssuer || !plate.plateReasonForIssue || !plate.toEmailAddress) {
```

Comment:
Why are we doing validation over here? Is this not validated in the validation schema?

Reply:
All the plate fields are optional in the validation schema..so in order for plate generation to be successful all the fields have to be sent from the FE